packages/astro/templates/actions.mjs

```diff
@@ -22,6 +24,16 @@ function toActionProxy(actionCallback = {}, aggregatedPath = '/_actions/') {
 					data,
 				}
 			};
+			action.safe.$$FORM_ACTION = function () {
```

Comment:
This part is very small and probably can't be moved out. So I don't see any issue with having this part.